### PR TITLE
Remove instructions around SDK bridging headers - no longer required!

### PR DESCRIPTION
**Pull request by coryalder**
As of the Sept 13 2017 SDK release, a bridging header is no longer required during SDK integration. This PR removes the instructions involving bridging headers from the documentation.

Note: We don't mention that you need to `import BuddyBuildSDK` at the top of each test that you want to record video on. This mirrors the ObjC documentation which also omits the required `#import <BuddyBuildSDK/BuddyBuildSDK.h>`, but perhaps we should mention both?

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://54-bb-docs.surge.sh

File: sdk/integration.adoc [hunk 1]
```diff
@@ -132,8 +119,7 @@ header file should look something like this.
 ----
 --
 
-We're interested in associated the class implementation file. Add this
-import to the class implementation file.
+Add this import to the class implementation (.m) file.
```
Comment (bbewan) [inline sdk/integration.adoc]: Filenames, and references to filenames via extensions, should be in monospace. So:
```
... implementation (`.m`) file.
```

**Comment by bbdocs** [comment]
See the website here: http://54-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://54-bb-docs.surge.sh

**Pull request merged**